### ISSUE TITLE
`npm WARN EBADENGINE Unsupported engine` with Node v18.0.0

### ISSUE THREAD

**Issue by darkred** [Yash-Singh1/eslint-plugin-userscripts#38]
I have recently updated Node to v18.0.0 (x64 on win11). ESLint is v8.14.0
Since then, whenever trying to install or update the package I'm getting the title warning, i.e.

```
> npm install eslint-plugin-userscripts --save-dev

npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'eslint-plugin-userscripts@0.2.2',
npm WARN EBADENGINE   required: { node: '>=14.0.0 <18.0.0' },
npm WARN EBADENGINE   current: { node: 'v18.0.0', npm: '8.6.0' }
npm WARN EBADENGINE }
```

The same occurs with `npm update`.

&nbsp;
The fix for this is would be to change:
https://github.com/Yash-Singh1/eslint-plugin-userscripts/blob/1878fc4d20d6d64bf96ad42ce5f257cc92866ef7/package.json#L41-L43
to e.g. `    "node": ">=14.0.0 <=18.0.0"`

&nbsp;
I would gladly make a PR instead of an issue, but I think I prefer to leave it to you to define the version range as you wish.

&nbsp;
PS. Thanks for this very useful plugin!

**Comment by Yash-Singh1** [comment]
Thanks for creating this issue! I just updated the package to `0.2.3`, which should support NodeJS v18.

**Comment by darkred** [comment]
You're welcome! Thanks for fixing this.

**Comment by darkred** [comment]
@Yash-Singh1 Greetings. The same occurs again now with Node v22.2.0 (as shown in the error) when running `npm update`:

```
npm warn EBADENGINE Unsupported engine {
npm warn EBADENGINE   package: 'eslint-plugin-userscripts@0.5.2',
npm warn EBADENGINE   required: { node: '>=18.0.0 <22.0.0' },
npm warn EBADENGINE   current: { node: 'v22.2.0', npm: '10.7.0' }
npm warn EBADENGINE }
```

**Comment by Yash-Singh1** [comment]
Hi @darkred, I just published release 0.5.3 with a fix, could you confirm that it now works?

**Comment by darkred** [comment]
Yes, it's working now, thanks again!
